include/aspect/postprocess/composition_BPL_statistics.h

```diff
@@ -0,0 +1,54 @@
+/*
+  Copyright (C) 2011, 2012 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__postprocess_composition_BPL_statistics_h
+#define __aspect__postprocess_composition_BPL_statistics_h
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+
+    /**
+     * A postprocessor that computes some statistics about the compositional
+     * fields, if any.
```

Comment:
Please change the description of the postprocessor to give the reader (me in this case :smile:) an idea what this postprocessor is supposed to do. Also no two postprocessors should do the same thing -- that would duplicate code that would need to be fixed in several places if a bug occurs. So I think it would be best to limit this postprocessor to the *new* output that you generate.